### ISSUE TITLE
Visible label and aria label do not match in step summaries

### ISSUE THREAD

**Issue by chriswales95** [localgovdrupal/localgov_step_by_step#104]
A client flagged that the step-by-step summary aria-label and button text don't match. This seems to fail WCAG 2.2 2.5.3 Label in Name: 
* https://www.w3.org/TR/WCAG22/#label-in-name
* https://www.w3.org/WAI/WCAG22/Understanding/label-in-name

SiteImprove flags it on the LocalGov demo site too: 
![Screenshot 2024-08-20 at 12 03 59](https://github.com/user-attachments/assets/7a10b081-d479-43e5-8778-638c8767247f)

This simple solution seems to be changing the aria label to include the button text at the start. I'll make a PR for this.

**Comment by chriswales95** [comment]
Closed since it duplicates #66.
